src/features/user/TreeMapper/Import/components/PlantingLocation.tsx

```diff
@@ -3,16 +3,13 @@ import { Controller, useForm, useFieldArray } from 'react-hook-form';
 import MaterialTextField from '../../../../common/InputTypes/MaterialTextField';
 import styles from '../Import.module.scss';
 import i18next from '../../../../../../i18n';
-import DateFnsUtils from '@date-io/date-fns';
-import { DatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
-import { ThemeProvider } from '@material-ui/styles';
 import materialTheme from '../../../../../theme/themeStyles';
 import { localeMapForDate } from '../../../../../utils/language/getLanguageName';
 import { useDropzone } from 'react-dropzone';
 import DeleteIcon from '../../../../../../public/assets/images/icons/manageProjects/Delete';
 import JSONInput from 'react-json-editor-ajrm';
 import locale from 'react-json-editor-ajrm/locale/en';
-import { MenuItem } from '@material-ui/core';
+import { MenuItem, ThemeProvider } from '@mui/material';
```

Comment:
Slight difference in font sizes and other font styles compared to earlier.

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r867877084

Reply:
Fixed in #1467